server/src/controllers/requestController.js

```diff
@@ -18,14 +14,27 @@ export default class requestController {
    * @returns {Array} request
    */
   static getAll(req, res) {
+    let filter = '';
+    let presentQuery = '';
+    if (req.query.category) {
+      req.query.category = req.query.category.toLowerCase();
```

Comment:
Can this be achieved with the aid of middleware preharps so as to use the controller to test only legitimate data. I also dislike the fact you are repeating your validation logic for category and status, this will bring down your maintainability. Please fix !!

Reply:
alright, I will fix it